shuler_lab_to_nwb/fiberphotometry_datainterface/metadata.yaml

```diff
@@ -0,0 +1,105 @@
+NWBFile:
+  session_description: Fiber photometry recordings.
+  session_start_time: "2020-01-01T12:30:00-08:00"
+  institution: Johns Hopkins University School of Medicine
+  lab: Shuler Lab
+  experimenter:
+    - Zhang, Shichen
+Subject:
+  species: Mus musculus
+  age: 20W
+  sex: U
+FiberPhotometry:
+  ExcitationSourcesTable:
+    description: The excitation sources table.
+    items:
+      - peak_wavelength: 475.0 # in nanometers
+        source_type: LED # LED or laser
+      - peak_wavelength: 560.0 # in nanometers
+        source_type: LED # LED or laser
+  PhotodetectorsTable:
+    description: The metadata for the photodetector.
+    items:
+      - peak_wavelength: 510.0 # in nanometers
+        type: CMOS camera
+  FluorophoresTable:
+    description: The metadata for the fluorophores. The coordinates are in unit meters relative to Bregma (AP, ML, DV).
+    items:
+      - label: GFP
+        location: Right hemisphere.
+        coordinates:
+          - 0.0008
+          - 0.0015
+          - 0.00233
```

Comment:
Just checking; are these are in CCF v3?

Reply:
Ahh so this is the IBL bregma-centric framework then

For consistency with other NWB files on DANDI, the best practice would be to define the absolute `x,y,z` values for electrodes and triangulated sorted units in the Allen CCFv3, and define other custom columns for alternative reference frameworks

Check out the IBL files on DANDI for examples of what this looks like (they used 3 atlases over there!) as well as some handy functions they have for easily mapping between the bregma-centric values to CCF https://int-brain-lab.github.io/iblenv/_autosummary/ibllib.atlas.atlas.html, specifically [the `xyz2ccf` function](https://int-brain-lab.github.io/iblenv/_autosummary/ibllib.atlas.atlas.html#ibllib.atlas.atlas.AllenAtlas.xyz2ccf)

Out of curiosity, have they expressed why they prefer that particular coordinate framework?

Reply:
no, but they seem to use it consistently across implants and injection sites.
Thanks for the reference function, I'll apply it myself and use the transformed values directly on the metadata. Then highlight in the instructions that the values should always be entered using the Allen CCFv3 reference.